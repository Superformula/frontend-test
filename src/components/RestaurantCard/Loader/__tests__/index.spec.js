import React from 'react';
import { shallow } from 'enzyme';
import { RestaurantCardLoader } from '../';

describe('RestaurantCardLoader', () => {
  it('should render without failing', () => {
    const wrapper = shallow(<RestaurantCardLoader />);
    expect(wrapper).toMatchSnapshot();
  });
});
