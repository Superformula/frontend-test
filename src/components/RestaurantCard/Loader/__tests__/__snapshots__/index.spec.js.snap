// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantCardLoader should render without failing 1`] = `
<Wrapper>
  <ImageWrapper>
    <SkeletonLoader />
  </ImageWrapper>
  <ContentWrapper>
    <Title>
      <SkeletonLoader />
    </Title>
    <GenericLabel>
      <SkeletonLoader />
    </GenericLabel>
  </ContentWrapper>
</Wrapper>
`;
